lib/internal/test_runner/runner.js

```diff
@@ -479,13 +485,13 @@ function run(options) {
   let postRun = () => root.postRun();
   let filesWatcher;
   if (watch) {
-    filesWatcher = watchFiles(testFiles, root, inspectPort, signal, testNamePatterns);
+    filesWatcher = watchFiles(testFiles, root, inspectPort, signal, testNamePatterns, bail);
```

Comment:
what should happen when running `--watch` with `--bail`?

Reply:
it works normally, it will keep quitting at first error as you edit the file

Reply:
what about other files? doe they stop running as well?